app/views/queue_items/index.html.haml

```diff
@@ -4,29 +4,35 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @items.each do |queue|
+        = form_tag(update_queue_path, method: :post) do
```

Comment:
`method: :post` is default for all Rails forms. Required action here would be to delete it :)